src/__experimental__/components/select/select.spec.js

```diff
@@ -109,6 +109,35 @@ describe('Select', () => {
     });
   });
 
+  it('renders a placeholder when disabled and readOnly are false', () => {
+    const wrapper = renderWrapper();
+    expect(wrapper.find(Input).props().placeholder).toEqual('Please Select...');
+  });
+
+  it('does not render a placeholder when disabled is true and readOnly is false', () => {
+    const props = { disabled: true, readOnly: false };
+    const wrapper = renderWrapper({ props });
+    expect(wrapper.find(Input).props().placeholder).toEqual('');
+  });
+
+  it('does not render a placeholder when disabled is false and readOnly is true', () => {
+    const props = { disabled: false, readOnly: true };
+    const wrapper = renderWrapper({ props });
+    expect(wrapper.find(Input).props().placeholder).toEqual('');
+  });
+
+  describe.each(
```

Comment:
Doesn't this test case with `describe.each` actually covers the 3 test cases above?

Reply:
Yeah. Ed showed me another way to do the tests and I left both ways in the code. My bad.